package.json

```diff
@@ -4,9 +4,6 @@
   "description": "a library of conditions that are useful for end-to-end tests",
   "main": "lib/src/index.js",
```

Comment:
Would you mind removing or changing `main` and `types` as well?

Reply:
Ah, good catch!